Makefile

```diff
@@ -96,8 +110,32 @@ build-esp32-ota:
 		-Zbuild-std=std,panic_abort \
 		--release \
 		target/xtensa-esp32-espidf/release/micro-rdk-server-esp32-ota.bin
+
+build-esp32c6-ota:
+	cargo +nightly espflash save-image \
+		--skip-update-check \
+		--package=micro-rdk-server \
+		--chip=esp32c6 \
+		--bin=micro-rdk-server-esp32 \
+		--partition-table=micro-rdk-server/esp32/partitions.csv \
+		--target=riscv32imac-esp-espidf \
+		-Zbuild-std=std,panic_abort \
+		--release \
+		target/riscv32imac-esp-espidf/release/micro-rdk-server-esp32-ota.bin
+
 build-qemu-bin:
-	cargo +esp espflash save-image --release --package micro-rdk-server --features qemu --merge --chip esp32 target/xtensa-esp32-espidf/release/qemu.bin -T micro-rdk-server/esp32/partitions.csv -s 8mb  --bin micro-rdk-server-esp32 --target=xtensa-esp32-espidf -Zbuild-std=std,panic_abort
+	cargo +esp espflash save-image \
```

Comment:
Drive-by: I linewrapped these, re-ordered them to be consistent with the other `build-*-bin` targets, and normalized flags to the long form.